asterius/src/Asterius/Builtins.hs

```diff
@@ -590,8 +590,13 @@ floatCBits =
        ( AsteriusEntitySymbol func_sym
        , generateRTSWrapper "floatCBits" func_sym param_vts ret_vts))
     [ ("isFloatNegativeZero", [F32], [I64])
+    , ("isDoubleNegativeZero", [F64], [I64])
```

Comment:
Are the newly added functions called via `foreign import ccall` somewhere? It seems they are only called via `foreign import javascript` instead, in which case, we don't need to put them into the builtins import; JSFFI handles the relevant import automatically. But if something in `base` has a `ccall` import then we can leave the stuff here.